### ISSUE TITLE
Document how to bind to specific host IP addresses

### ISSUE THREAD

**Issue by benbucksch** [traefik/traefik#5059]
I would like to bind an instance of traefik to given specific host IP addresses (IPv4 and/or IPv6) and not to all interfaces. I did not see a configuration option documented for this.

There is #1193 , which states that `address = "1.2.3.4:80"` should work, but does not state how to specify multiple addresses, how to specify IPv6 addresses with `::` syntax etc..

The correct solution is not even in traefik.toml at all, but in the docker run command or docker-compose file. E.g.
```
docker run -d \
  ...
  -p 1.2.3.4:80:80 \
  -p 1.2.3.5:80:80 \
  -p 1.2.3.4:443:443 \
  -p 1.2.3.5:443:443 \
  ...
  traefik
```
works fine for me.

Could you please document it on https://docs.traefik.io/configuration/entrypoints/ ?

This would help others. It took me hours to figure that out. It may appear to be obvious to traefik developers and power users, and may appear offtopic to traefik, but this is a common problem for people who start with traefik and docker.

**Comment by geraldcroes** [comment]
Hi and thanks for your feedback! Improving the documentation is always a good idea.
Since you struggled with it recently, do you think you could work on improving that part and make a PR?
That would definitely help us (and others)

**Comment by benbucksch** [comment]
In which repo and file is this document? https://docs.traefik.io/configuration/entrypoints/

**Comment by ldez** [comment]
the documentation of the v1 is here https://github.com/containous/traefik/tree/v1.7/docs

**Comment by SantoDE** [comment]
@benbucksch the exact file you're looking for is in:

https://github.com/containous/traefik/blob/v1.7/docs/configuration/entrypoints.md

**Comment by darkermatter** [comment]
The solution is correct... for Docker.

What if you don't use Docker?

**Comment by ldez** [comment]
Related to #4961

**Comment by s0rcerer88** [comment]
Maybe it could be correct if you only want to bind all ip addresses to an entrypoint, but, what happens if you want to define separated entrypoints for each ip? Is it possible using traefik as a docker container? I've not found any related info.

**Comment by dduportal** [comment]
Hi @s0rcerer88 , you might want to join our Community Forum at http://community.containo.us as we use Github issues for bugs and features requests.

To get information about how to use Traefik as a docker container, you can start with the documentation: https://docs.traefik.io/getting-started/install-traefik/ and https://docs.traefik.io/getting-started/quick-start/.

By the way, you can totally create one entrypoint per couple IP/port on your machine. As Traefik v2 binds routers to all entrypoints by default (unless restriction) as explained in <https://docs.traefik.io/routing/routers/#entrypoints>, there is no problem here.

**Comment by s0rcerer88** [comment]
Hi @dduportal ,
I've been searching at documentation and trying some things, but at least for me, is not possible to bind separated ips for each endpoint. When you try to do this, log says that traefik can't bind that ip, and if you start traefik binded to the host, you cant access anything, including the dashboard.

Can you guide me to some kind of solution please?

Thank you!!

**Comment by dduportal** [comment]
Hi @s0rcerer88 can you switch to the community forum please?

**Comment by jbdoumenjou** [comment]
fixed by #6483
